### DISCUSSION TITLE
setImage of multiaction

### DISCUSSION THREAD

**Discussion by mekhaelt** [elgatosf/elgato#54] [Help] [answered]
Is there anyway to change the image of a multi-action button. I tried using setImage, but it wasn't working

**Comment by ChekTek** [comment, marked as answer]
This is not possible at the moment. I believe this feature would require some UI for the user to also allow a certain action to take priority for setting the image, since multiple actions are on one key. 

I will share this with the team to get a better idea of what the feature could look like.
